lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1650,9 +1692,16 @@ def cmd_sessions(*args)
         end
       end
     when 'killall'
-      print_status("Killing all sessions...")
-      framework.sessions.each_sorted.reverse_each do |s|
-        session = framework.sessions.get(s)
+      if matching_sessions
+        print_status('Killing matching sessions...')
+        print_line
+        print_status(Serializer::ReadableText.dump_sessions(framework, show_active: show_active, show_inactive: show_inactive, show_extended: show_extended, verbose: verbose, sessions: matching_sessions))
```

Comment:
Same here as below, just regular print for tables, my bad
```suggestion
        print(Serializer::ReadableText.dump_sessions(framework, show_active: show_active, show_inactive: show_inactive, show_extended: show_extended, verbose: verbose, sessions: matching_sessions))
```